Formula/espflash.rb

```diff
@@ -0,0 +1,36 @@
+class Espflash < Formula
+  desc "Serial flasher utility for Espressif SoCs and modules based on esptool.py"
+  homepage "https://github.com/esp-rs/espflash"
+  url "https://github.com/esp-rs/espflash/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "01b65bdd7c22b422d18db6cf186cb788151f0ca0eda9533ff9478aaf7255a2cf"
+  license any_of: ["MIT", "Apache-2.0"]
+
+  depends_on "rust" => :build
+
+  on_macos do
+    depends_on "libuv" => :build
+  end
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "systemd" # for libudev
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "espflash")
+  end
+
+  test do
+    resource "homebrew-testdata" do
+      url "https://raw.githubusercontent.com/esp-rs/espflash/v2.0.0/espflash/tests/resources/esp32_hal_blinky"
+      sha256 "e869d05c3b4e670bb9d9f2ef304e7f32371bc4a2cff6cb90461546dfa3380473"
+    end
+
+    testpath.install resource("homebrew-testdata")
```

Comment:
```suggestion
    stable.stage testpath
    testpath.install "espflash/tests/resources/esp32_hal_blinky"
```
We don't need to declare a resource that's essentially a duplicate of the stable tarball.